src/partner-access/dto/create-access-code.dto.ts

```diff
@@ -0,0 +1,23 @@
+import { IsNotEmpty, IsBoolean, IsInt, IsUUID } from 'class-validator';
+
+export class CreateAccessCodeDto {
```

Comment:
should this class be `PartnerAccessDto` instead of `CreateAccessCodeDto`? the `createAccessCode` is a function, not the overall model